tests/unit/CMakeLists.txt

```diff
@@ -123,7 +123,7 @@ add_unit_test(query_serialization_property_value.cpp)
 target_link_libraries(${test_prefix}query_serialization_property_value mg-query)
 
 add_unit_test(query_streams.cpp)
-target_link_libraries(${test_prefix}query_streams mg-query kafka-mock)
+target_link_libraries(${test_prefix}query_streams kafka-mock mg-query)
```

Comment:
Please don't ask...

Reply:
I will have a look at this, because usually these are the forerunner of some more nasty bugs.

Reply:
https://stackoverflow.com/questions/69889858/how-can-reordering-the-linked-libraries-make-multiple-definition-work